app/Filament/Resources/SettingResource/Pages/ManageSettings.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Filament\Resources\SettingResource\Pages;
+
+use App\Filament\Resources\SettingResource;
+use Filament\Resources\Pages\ManageRecords;
+use Filament\Actions\Action;
+use Illuminate\Database\QueryException;
+use App\Models\Setting;
+use App\Traits\Commands\EnvironmentWriterTrait;
+use Illuminate\Support\Facades\Artisan;
+
+class ManageSettings extends ManageRecords
+{
+    use EnvironmentWriterTrait;
+
+    protected static string $resource = SettingResource::class;
+    protected array $variables = [];
+
+    protected function getHeaderActions(): array
+    {
+        return [
+            Action::make('apply')
+                ->label('Apply Settings')
+                ->requiresConfirmation()
+                ->action($this->setConfigFromDatabase()),
+        ];
+    }
+
+    protected function setConfigFromDatabase(): void
+    {
+        try {
+            $settings = Setting::pluck('value', 'key')->toArray();
+        } catch (QueryException $exception) {
+            return;
+        }
+
+        foreach ($settings as $key => $value) {
+            $this->variables[$key] = $value;
+        }
+
+        $this->writeToEnvironment($this->variables);
```

Comment:
Settings are not written to the .env file.
Please see https://github.com/pelican-dev/panel/pull/259#discussion_r1611289143

Reply:
Most settings are. Like the APP_NAME and the mail settings etc. I could not get it to work with the config and the settings provider so I thought this was easier anyway since almost all of the settings end up in the .env file anyway, but I have a meeting with Lance about this PR today and I will discuss it with him how he would like to have it work.